src/styles/modules/_font-icon.scss

```diff
@@ -1,5 +1,5 @@
-[class^="dc-font-icon-"] {
-    color: $dc-blue40;
+[class^="#{$dc-prefix}-font-icon-"] {
```

Comment:
@gabrielhl please remove the dcPrefix variable usage, is deprecated.... I will remove it from the variables so it will simply raise an error in the future...